models/DataObject/ClassDefinition/Layout.php

```diff
@@ -80,7 +80,7 @@ class Layout
     /**
      * @var array
      */
-    public $childs = [];
+    public $children = [];
```

Comment:
Is this a BC break? This class or property doesn't seem to be `@internal`.

Reply:
I think these attributes should be internal. It should be used the getter and setter. There are only public to send it to JavaScript. Can we make it internal in 10.3 or is this also a BC for you?
Otherwise we must live with these deprecations till v11 or we can try to make both attributes. Maybe one tree with childs properties and one tree with children properties.

Reply:
What if we make it a reference in the constructor? 
Similar to: https://github.com/pimcore/pimcore/blob/6.9/models/Document/Editable.php#L118

Reply:
But this reference then no longer works in the json response to javascript right? I can imagine that this json will be quite large with complex layouts. But I can test it out.

Reply:
The reference itself not, but is that necessary? The data is after JSON serialization in both properties, so that should be enough to work in the UI.

Reply:
I changed it. But as i feared the response will be much bigger:
For example:
Before: 6,7 kB
After: 85,2 kB

But if that doesn't matter, we can leave it that way. An alternative would be a BC layer in PHP with __set and __get methods

Reply:
I create the BC layer in PHP with __set/__get methods and create for JS a childs tree. So the response is much smaller
For example:
Before: 6,7 kB
After: 14,4 kB